orchestrators/FinanceOperations/src/DTOs/RuleContext.php

```
@@ -0,0 +1,135 @@
+    public function __construct(
+        private string $tenantId,
+        private ?string $budgetId = null,
+        private ?string $amount = null,
+        private ?string $costCenterId = null,
+        private array $costCenterIds = [],
+        private ?string $periodId = null,
+        private ?string $subledgerType = null,
+        private array $transactionTypes = [],
+    ) {
+        if (trim($tenantId) === '') {
+            throw new \InvalidArgumentException('Tenant ID cannot be empty');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '26', column '23'). (undefined)

(MissingImport)